app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
+
+
+def create_person_list(people: list[dict]) -> list:
+    for person in people:
+        _ = Person(name=person.get("name"), age=person.get("age"))
```

Comment:
There is no need to process the None output of creating a person, you can omit that and leave just initialization.